modules/dashboard/js/dashboard-helper.js

```diff
@@ -13,63 +15,201 @@ var siteColours = [
 // Colours for the recruitment bar chart: breakdown by gender
 var genderColours = ['#2FA4E7', '#1C70B6'];
 
-$(document).ready(function () {
-    "use strict";
-    // Turn on the tooltip for the progress bar - shows total male and female registered candidates
-    $('.progress-bar').tooltip();
+// Turn on the tooltip for the progress bar - shows total male and female registered candidates
+$('.progress-bar').tooltip();
 
-    // Make dashboard panels collapsible
-    $('.panel-heading span.clickable').on("click", function () {
-        if ($(this).hasClass('panel-collapsed')) {
-            // expand the panel
-            $(this).parents('.panel').find('.panel-body').slideDown();
-            $(this).removeClass('panel-collapsed');
-            $(this).removeClass('glyphicon-chevron-down').addClass('glyphicon-chevron-up');
-        } else {
-            // collapse the panel
-            $(this).parents('.panel').find('.panel-body').slideUp();
-            $(this).addClass('panel-collapsed');
-            $(this).removeClass('glyphicon-chevron-up').addClass('glyphicon-chevron-down');
-        }
+// Make dashboard panels collapsible
+$('.panel-heading span.clickable').on("click", function () {
+    if ($(this).hasClass('panel-collapsed')) {
+        // expand the panel
+        $(this).parents('.panel').find('.panel-body').slideDown();
+        $(this).removeClass('panel-collapsed');
+        $(this).removeClass('glyphicon-chevron-down').addClass('glyphicon-chevron-up');
+    } else {
+        // collapse the panel
+        $(this).parents('.panel').find('.panel-body').slideUp();
+        $(this).addClass('panel-collapsed');
+        $(this).removeClass('glyphicon-chevron-up').addClass('glyphicon-chevron-down');
+    }
+});
+
+// Open the appropriate charts from the "views" dropdown menus
+$(".dropdown-menu a").click(function () {
+    $(this).parent().siblings().removeClass("active");
+    $(this).parent().addClass("active");
```

Comment:
I'm not particularly experienced with JQuery, but lines 38-39 seemed a bit convoluted. I looked up the syntax being used and found the following suggested alternative: https://stackoverflow.com/questions/3972944/jquery-removeclass-on-parent-sibling-child

which would be the following here I believe: 
$(".dropdown-menu").click(function () {
     $(this).parent().addClass('active').siblings().removeClass('active');
});